backend/src/Controller/Image/ImageController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Controller\Image;
```

Comment:
we gonna need controller for APIs related to images, like get all images for specific conditions